po/it.po

```diff
@@ -263,13 +265,13 @@ msgstr "La finestra principale con la playlist visualizzata"
 #. Translators: This is an icon name, don't translate!
 #: data/io.github.celluloid_player.Celluloid.desktop.in:8
 msgid "io.github.celluloid_player.Celluloid"
-msgstr ""
+msgstr "io.github.celluloid_player.Celluloid"
 
 #. Translators: Search terms to find this application. Don't translate the
 #. semicolons! The list MUST also end with a semicolon!
 #: data/io.github.celluloid_player.Celluloid.desktop.in:16
 msgid "Video;Movie;Film;Clip;Series;Player;DVD;TV;Disc;Album;Music;GNOME;mpv;"
-msgstr ""
+msgstr "Video, film, film, clip; Serie; Player, DVD, TV, Disco; Album; Musica; GNOME; mpv;"
```

Comment:
Only semicolons should be used to separate entries. Also, there's no need to list identical entries twice.

```suggestion
msgstr "Video;film;clip;Serie;Player;DVD;TV;Disco;Album;Musica;GNOME;mpv;"
```